### PR TITLE
release_schema: add organization/details/classifications

### PR DESCRIPTION
**Pull request by odscjen**
addresses #1622 

once this is merged need to address the changes to the extension as per https://github.com/open-contracting/standard/issues/1622#issuecomment-1581756030

### CODE REVIEW

File: docs/history/changelog.md [hunk 1]
```diff
@@ -210,6 +210,7 @@ Per the [normative and non-normative content and changes policy](../governance/n
   * [#1490](https://github.com/open-contracting/standard/pull/1490) `SimpleIdentifier`
   * [#1519](https://github.com/open-contracting/standard/pull/1519) `Value.amountNet` and `Value.amountGross`
   * [#1490](https://github.com/open-contracting/standard/pull/1490) `contracts.identifiers`
+  * [#1679](https://github.com/open-contracting/standard/pull/1679) `Organization.details.classifications` merged from Organization classifications extension
```
Comment (duncandewhurst) [inline docs/history/changelog.md]: The changelog doesn't mention the other extensions from which fields were merged (submission terms and pagination).

```suggestion
  * [#1679](https://github.com/open-contracting/standard/pull/1679) `Organization.details.classifications`
```

**Review by duncandewhurst — COMMENTED** [review]
Should the additions to the `itemClassificationScheme` codelist also be merged?

**Comment by odscjen** [comment]
> Should the additions to the itemClassificationScheme codelist also be merged?

I had initially thought no because they're all in the 1.1 extension branch too, but now I realise you're right, I'll move them over

File: docs/history/changelog.md [hunk 2]
```diff
@@ -84,6 +84,11 @@ Per the [normative and non-normative content and changes policy](../governance/n
     * 'COFOG'
   * [#1452](https://github.com/open-contracting/standard/pull/1452) Add 'proClass'.
   * [#1637](https://github.com/open-contracting/standard/pull/1637) Add 'sdg' and 'sdgTarget'.
+  * [#1679](https://github.com/open-contracting/standard/pull/1679) Add codes from the organization classification extension:
+    * 'TED_CA_TYPE'
+    * 'TED_CE_ACTIVITY'
+    * 'eu-buyer-contracting-type'
+    * 'eu-main-activity'
```
Comment (duncandewhurst) [inline docs/history/changelog.md]: Suggest combining with the existing changelog entry for PR #1218

**Review by duncandewhurst — CHANGES_REQUESTED** [review]
Happy for you to request a review from James once the changelog comment is addressed.

File: schema/release-schema.json [hunk 3]
```diff
@@ -1429,6 +1429,17 @@
               ],
               "codelist": "partyScale.csv",
               "openCodelist": false
+            },
+            "classifications": {
+              "title": "Organization classifications",
+              "description": "The classifications that categorize the organization",
```
Comment (jpmckinney) [inline schema/release-schema.json]:
```suggestion
              "description": "The classifications that categorize the organization.",
```

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**